metadata.json

```
@@ -48,7 +48,7 @@
     },
     {
       "name": "puppetlabs/apt",
-      "version_requirement": ">= 2.2.0 < 7.0.0"
+      "version_requirement": ">= 2.2.0 < 7.1.0"
```

Usually we follow semver and allow the entire major version
```suggestion
      "version_requirement": ">= 2.2.0 < 8.0.0"
```

---

done